### PR TITLE
SQLite generator fix

### PR DESCRIPTION
**Pull request by Evolution0**
I'm kinda surprised this wasn't noticed really as it would cause it to not run at all on a fresh install, the generated SQLite script would have a comma appended to the last attributes of specific tables; namely those generated from lists of attributes.

I tweaked the if/else bit to sort that out as well as added some formatting (doesn't effect execution) so that the SQLite prints better (for debugging)

I've also started trying to get the code up to PEP8 standard, some lines are simply too long and I don't want to adhere to PEP8 bad enough to mangle the code so bits here and there will be left as-is.

Basically anything that improves readability or simplifies the code without affecting its behavior I'll do.

Tested and working on Python 3.5.2 & 3.6.0 under Windows 10 using QT 5.8

None of the changes are drastic enough to effect Python 3.4.

### CODE REVIEW

**Comment by rachmadaniHaryono** [comment]
Hi @Evolution0, can you separate the fix and refactor/cosmetic pr? Also do you have unit test for the Fix? 

Also @Pewpews didn't make any setting for code linter, but following most of the code, the line length should be longer than 80. Several method name also didn't follow flake8 because it follow the qt c++ convention.

**Comment by Evolution0** [comment]
Yeah the line length is unavoidable really due to its structure and I can understand sticking to QTs code style.

I'll revert, isolate the cosmetic from the actual fix, and squash my commits.

**Comment by rachmadaniHaryono** [comment]
Maybe if possible join gitter.im chat and discuss it there?

**Comment by rachmadaniHaryono** [comment]
Why not using ', '.join(col_list) ?

**Comment by Evolution0** [comment]
Honestly I didn't want to get out of the scope of the fix, but since I would probably do that later anyway should it be added?

**Comment by rachmadaniHaryono** [comment]
imo it is not out of scope. on the original code there is too much if-else which can be replaced with single line `sql += ', '.join(col_list)`. but i don't know much about sql command, so i don't know if it is correct.

what is your opinion @Pewpews ?

File: version/database/db.py [hunk 1]
```diff
@@ -264,7 +274,7 @@ def series_list_map_sql(cols=False):
 
 STRUCTURE_SCRIPT = series_ns_sql()+cirlce_sql()+artist_sql()+series_sql()+chapters_sql()+namespaces_sql()+tags_sql()+tags_mappings_sql()+\
     series_tags_mappings_sql()+hashes_sql()+list_sql()+series_list_map_sql()
-
+print(STRUCTURE_SCRIPT)
```
Comment (rachmadaniHaryono) [inline version/database/db.py]: maybe log_d instead of print

Comment (Evolution0) [reply version/database/db.py]: Yeah forgot about that.

**Comment by twiddli** [comment]
It's definitely correct and should be used instead of the `if...else..` statements to join the queries together. It's less error_prone.

I come from C++ with no such syntactic sugar, so I had totally forgotten about `.join`.

**Comment by rachmadaniHaryono** [comment]
i'm okay with adding empty new line for readibility.

but maybe add tab or comma on existing code can be do it later? e.g. 

https://github.com/Pewpews/happypanda/pull/173/commits/9ec1b3ca8cb3b2be1830b35d34cb452a3d413a59#diff-1957902c318983068100f9127a285cbeR80

https://github.com/Pewpews/happypanda/pull/173/commits/9ec1b3ca8cb3b2be1830b35d34cb452a3d413a59#diff-1957902c318983068100f9127a285cbeR145

**Comment by Evolution0** [comment]
Should be good to go now.
